web/src/index.js

```diff
@@ -1,24 +1,40 @@
 import { fetchNFT, fetchActiveAd } from '../../utils/networking'
+import { formats, defaultFormat } from '../../utils/formats';
 
 class Zesty extends HTMLElement {
     constructor() {
         super();
         this.adSpace = "";
         this.creator = "";
+        this.adFormat = defaultFormat;
         this.width = "100%";
         this.height = "100%";
         this.shadow = this.attachShadow({mode: 'open'});
     }
 
     connectedCallback() {
+        this.style.cursor = "pointer";
+        
         this.adSpace = this.getAttribute("adspace");
         this.creator = this.getAttribute("creator");
+        this.adFormat = this.hasAttribute("adformat") ? this.getAttribute("adformat") : this.adFormat;
```

Comment:
Is `format` a reserved variable? Would like to start removing the word `ad` from across Zesty.

Reply:
Don't think it is. I'll address this in a separate PR that'll just go through and update naming.